doc/source/installation/azure_cloud.rst

```diff
@@ -0,0 +1,67 @@
+.. _azure_cloud:
+
+===============================================
+Deploying |Percona Server| on Azure Marketplace
+===============================================
+
+Microsoft Azure image of |Percona Server| is hosted publicly as `Azure Marketplace app <https://azuremarketplace.microsoft.com/en-us/marketplace/apps/percona.percona-server?tab=Overview>`_:
```

Comment:
I think `app` is not needed here, in fact full name is `Microsoft Azure Marketplace`

Reply:
oh, ok, thanks